package/batocera/emulationstation/batocera-es-system/es_features.yml

```diff
@@ -26,14 +26,15 @@ shared:
         "6": 6
     secondinstance:
       submenu:     LATENCY REDUCTION
-      prompt:      RUN-AHEAD USE SECOND INSTANCE
+      prompt:      USE SECOND INSTANCE FOR RUN-AHEAD
```

Comment:
Question: will that require new translations for every language? If so, we'd better wait for v34